package.json

```diff
@@ -22,55 +22,54 @@
     "lint": "run-s check-types run-lint",
     "build": "tsc",
     "prepublishOnly": "tsc",
-    "test": "nyc ts-node -r dotenv-safe/config ./test/index.ts"
+    "test": "nyc ts-node ./test/index.ts"
   },
   "dependencies": {
     "form-data": "^4.0.0",
     "lodash.clonedeep": "^4.5.0",
     "lodash.set": "^4.3.2"
   },
   "peerDependencies": {
-    "@apollo/gateway": "^2.0.1",
+    "@apollo/gateway": "^2.0.4",
     "apollo-server-env": "^4.2.1",
     "apollo-server-types": "^3.5.2",
-    "graphql-upload": "^13.0.0"
+    "graphql-upload": "^13.0.0",
+    "node-fetch": "^3.2.5"
```

Comment:
I was wondering. What do you think about this as dep and not a peer dep?

Reply:
Quick question, is there any plan to upgrade this to the latest version?
https://www.npmjs.com/package/graphql-upload

Reply:
@cabelitos could be, I kept it as before, but I think it may be worth to have as dep as it's only use directly (unlike `@apollo/gateway` that is use by our users).

@douglas-wang it's already 15.0.1 at the devDependency, but I'll move this to a simple direct dependency.